pkg/storaged/crypto-keyslots.jsx

```diff
@@ -253,6 +256,191 @@ export function init_existing_passphrase(block, just_type, callback) {
     };
 }
 
+/* Getting the system ready for NBDE on the root filesystem.
+
+   We need the clevis module in the initrd.  If it is not there, the
+   clevis-dracut package should be installed and the initrd needs to
+   be regenerated.  We do this only after the user has agreed to it.
+
+   The kernel command line needs to have rd.neednet=1 in it.  We just
+   do this unconditionally because it's so fast.
+*/
+
+function ensure_package_installed(steps, progress, package_name) {
+    function status_callback(progress) {
+        return p => {
+            let text = null;
+            if (p.waiting) {
+                text = _("Waiting for other software management operations to finish");
+            } else if (p.package) {
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17796-20221124-103059-3f8f659c-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)